include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -111,7 +118,9 @@ struct CShuffleEpilogue
         }
     }
 
-    template <typename ODramWindowTmp, typename OAccTile>
+    template <typename ODramWindowTmp,
+              typename OAccTile,
+              memory_operation_enum out_memory_data_op = memory_operation_enum::set>
     CK_TILE_DEVICE auto operator()(ODramWindowTmp& o_dram_window_tmp, OAccTile& o_acc_tile)
```

Comment:
I think if you could add third function parameter `memory_operation_enum o_mem_data_op = out_memory_data_op` ? Then you wouldn't have to pass all template parameters, but just pass memory op if you need different one than default.

Reply:
I think it is not possible to pass it because it is enum. Then you cannot compare object from argument in `if constexpr`

Reply:
You're right. What about having a `store_tile` API which get's as as last paramter memory operation enum ? And for `set` it will do _store_ while for `atomic_add` and others it will do _update_ ?